plugins/org.eclipse.thym.ui/plugin.xml

```diff
@@ -304,4 +304,22 @@
             name="Install Cordova Requirements">
       </cheatsheet>
    </extension>
+   <extension
+         point="org.eclipse.ui.decorators">
+      <decorator
+            id="org.eclipse.thym.ui.wwwDecorator"
+            label="%WWWDecorator.name"
+            class="org.eclipse.thym.ui.decorators.WWWFolderDecorator"
+            adaptable="true"
+            lightweight="true"
+            location="TOP_LEFT"
+            state="false">
```

Comment:
I think this decorator is not very invasive so it  is OK to enable this decorator by default.